components-core/src/main/java/com/airwallex/android/core/ActionComponentProviderType.kt

```diff
@@ -1,5 +1,15 @@
 package com.airwallex.android.core
 
-enum class ActionComponentProviderType {
-    CARD, REDIRECT, WECHATPAY, GOOGLEPAY
-}
+enum class ActionComponentProviderType(val value: String) {
+    CARD("card"),
+    REDIRECT("redirect"),
+    WECHATPAY("wechatpay"),
+    GOOGLEPAY("googlepay");
+
+    companion object {
+        fun fromValue(value: String?): ActionComponentProviderType {
+            return ActionComponentProviderType.values()
+                .firstOrNull { it.value == value } ?: REDIRECT
```

Comment:
Why do we make the fallback value as `REDIRECT`? Will `null` be better since no component/provider should handle an unknown payment method type?

Reply:
Because there are so many types of redirects, such as Alipay, FPS, etc., it is difficult to list all of them.